alarm/kodi-rbp-git/PKGBUILD

```diff
@@ -24,54 +26,37 @@ makedepends=('hicolor-icon-theme' 'fribidi' 'lzo' 'smbclient' 'libtiff' 'libva'
              'libmpeg2' 'libmad' 'libmodplug' 'jasper' 'rtmpdump' 'unzip' 'xorg-xdpyinfo' 'libbluray'
              'libnfs' 'afpfs-ng' 'avahi' 'bluez-libs' 'tinyxml' 'raspberrypi-firmware' 'libcec-rpi'
              'libplist' 'swig' 'taglib' 'libxslt' 'shairplay' 'boost' 'cmake' 'gperf' 'nasm' 'zip'
-             'upower' 'git' 'autoconf' 'java-environment' 'libcrossguid-git' 'dcadec' 'libpulse' 'cwiid')
-source=("xbmc-$_tag.tar.gz::https://github.com/xbmc/xbmc/archive/$_tag.tar.gz"
+             'upower' 'git' 'autoconf' 'java-environment' 'libcrossguid-git' 'dcadec' 'libpulse' 'cwiid'
+             'libfmt-git' 'rapidjson-git'
+)
+source=("$_gitname::git+https://github.com/xbmc/xbmc.git"
         'kodi.service'
         '99-kodi.rules'
-        'https://github.com/popcornmix/xbmc/commit/0c320b6cdd4fb409be45008e6b9042463d54b742.patch'
-        'https://github.com/popcornmix/xbmc/commit/4d983105d7fd65b1d92f2ae2602e6e1cdcaddbe0.patch'
+        'kodi.service'
         'polkit.rules')
-sha256sums=('a614128c065d561fccd4167f25eddbad3875328f1e81f627ffe3d0b17958aa07'
+
+sha256sums=('SKIP'
             '5235068d5800d69f0287087815990e7fe8d6572733d60c8800546d35f608e87f'
             'b31570f95654434b01fd8531612fbb6be77cbc1c519dd60f92feae26eb160f3d'
-            '813f8c622c341eefb33774115199d2abe9c189cf2ce52e79719dbd42d48a1274'
-            'd830c010ead152bc9a9ba6b812c985d6e649bda7480fec9b3c451f5ea03ba792'
+            '5235068d5800d69f0287087815990e7fe8d6572733d60c8800546d35f608e87f'
             '9ea592205023ba861603d74b63cdb73126c56372a366dc4cb7beb379073cbb96')
 
-prepare() {
-  cd "$srcdir/xbmc-$_tag"
 
-  # ffmpeg: Automatic switch to software decode for GMC with more than one warp point
-  patch -Np1 -i "$srcdir/0c320b6cdd4fb409be45008e6b9042463d54b742.patch"
-  # fix for files with GMC
-  patch -Np1 -i "$srcdir/4d983105d7fd65b1d92f2ae2602e6e1cdcaddbe0.patch"
+pkgver() {
+  cd "$srcdir/$_gitname"
+  git log -1 --date=short --format="%cd.%h" | tr -d '-'
+}
+
 
-  [[ -d kodi-build ]] && rm -rf kodi-build
+prepare() {
+  rm -rf $srcdir/kodi-build
   mkdir $srcdir/kodi-build
 }
 
 build() {
-  cd "kodi-build"
-
-  # -DCPU=arm1176jzf-s for rpi1
-  # -DCPU=cortex-a7 for rpi2
-  # -DCPU=cortex-a53 for rpi3
-  
-  [[ $CARCH == "armv6h" ]] && _CPU=arm1176jzf-s
-  [[ $CARCH == "armv7h" ]] && _CPU=cortex-a7
```

Comment:
you should leave this stuff in along with the WITH_CPU on the cmake or you'll break distcc building